sky/clouds/kubernetes.py

```diff
@@ -64,6 +64,9 @@ class Kubernetes(clouds.Cloud):
                                                              'tiers are not '
                                                              'supported in '
                                                              'Kubernetes.',
+        clouds.CloudImplementationFeatures.HOST_CONTROLLERS: 'Kubernetes can '
```

Comment:
Might be good to have a comment for why it cannot host controller at the moment.